### DISCUSSION TITLE
Values should be converted to the unit for the overpower feature

### DISCUSSION THREAD

**Discussion by Toniob** [jmcollin78/versatile_thermostat#634] [Ideas]
Hello,
It is possible for you to know what is the current unit of a sensor ? If so, would it be ok to convert the values to the same unit ? We could say to put everything in W or VA ? If another value (for exemple, the maximum power for my network is 6kVA) is not in Watts, we convert it. For kW or kVA, we have to multiple the value by 1000. If the value is in A, we need the Tension to put it in W. That way, no issue to check if all units are the same. Do you think it’s possible ?

**Comment by jmcollin78** [comment]
Hello @Toniob ,

Thank you for your suggestion, all is possible with time and money 😄 . And time is exactly what is missing.
The drawback I see is that you set power in the configuration of the Vtherm and the unit can be determined at startup time (and not at config time). So I'm not sure it is possible simply.
